docs/glossary.md

```diff
@@ -0,0 +1,41 @@
+<a id="top"></a>
+
+# Glossary of Terms
+
+
+| MLRun Terms              | Definition                                                                                                                                                                                    |
+| -------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| FeatureSet               | Group of features that are ingested together and stored in logical group                                                                                                                      |
+| FeatureVector            | Combination of multiple FeatureSets with ability to join, filter, etc.                                                                                                                        |
+| MLRun Function           | Abstraction that allows for running Python code easily in local environment and on top of K8s                                                                                                 |
+| MLRun Marketplace        | Collection of pre-built MLRun functions avilable for usage                                                                                                                                    |
+| MLRun Project            | Logical separation of resources - includes model, jobs, real-time functions, datasets, feature sets, feature vectors, etc.                                                                    |
```

Comment:
a logical container for all the work on a particular activity/application which include functions, workflow, artifacts, secrets, and more. and can be assigned to a specific group of users